all-the-icons.el

```diff
@@ -526,6 +543,12 @@
 ;;   Functions Start
 ;; ====================
 
+(defun* all-the-icons-is-repo (path)
+  "Check if PATH is a repository."
+  (dolist (repo all-the-icons-mode-repositories)
```

Comment:
Emacs already has a `vc-mode` built-in that comes with a custom called `vc-directory-exclusion-list`, can you try reusing it in your `all-the-icons-is-repo`, how about this:

```suggestion
  (dolist (repo vc-directory-exclusion-list)
```

Don't forget to load this variable somewhere in your file as well.